src/constants.ts

```diff
@@ -5,7 +5,7 @@ export const MAIN_MIN_WIDTH = DEFAULT_WIDTH - SIDEBAR_WIDTH + 20;
 export const SCREENSHOT_WIDTH = 1040;
 export const SCREENSHOT_HEIGHT = 1248;
 export const LIMIT_OF_ZIP_SITES_PER_USER = 5;
-export const LIMIT_OF_PROMPTS_PER_USER = 200;
+export const LIMIT_OF_PROMPTS_PER_USER = 1000;
```

Comment:
The actual limit for regular users is 200. We get 1000 prompts for debugging purposes. Let's change it back to 200.

```suggestion
export const LIMIT_OF_PROMPTS_PER_USER = 200;
```